README.md

```diff
@@ -12,14 +12,34 @@
 </p><br>
 
 ## Installation
-You can install q2-fondue using mamba in a conda environment of its own (option 1) or in an existing QIIME 2 environment (option 2) by following the steps described below. The current q2-fondue version supports QIIME 2 **v2021.4** or higher. To install q2-fondue with a version <= 2023.7 see section ["Installing q2-fondue with version <= 2023.7"](#legacy). Alternatively, a minimal Docker image is available to run q2-fondue v2023.7 (option 3 below - note other Docker image q2-fondue versions are work in progress).
+There are multiple options to install q2-fondue - each targeted towards different needs. You can install q2-fondue (**v2024.5** or higher):
+* as part of the QIIME 2 metagenome distribution if you want to use it in the QIIME 2 metagenome universe (see [option 1](#option-1-install-q2-fondue-with-qiime-2-metagenome-distribution)).
```

Comment:
the bulleted list is not necessary here as you have more detailed instructions below.

Reply:
In contrast to the more detailed instructions below, the bulleted list highlights the different use cases for analyses starting with q2-fondue. As a user, I always appreciate such overviews - so I am in favour of keeping this as is, unless you strongly disagree.

Reply:
as the detailed instructions are quite short the bullet points just feel repetitive, and increase the maintenance. So I do not strongly disagree, but feel they are unnecessary.